style.css

```diff
@@ -0,0 +1,746 @@
+
+.magazine-section {
+  width: 85vw;
+  margin: 0 auto;
+}
+
+.cat-and-section {
+  display: flex;
+  justify-content: space-between;
+  width: 50%;
+  align-items: center;
+}
+
+.mobile-display {
+  display: none;
+}
+
+.cat-and-section a {
+  text-decoration: none;
+  color: #f72210;
+  font-weight: 800;
+  font-size: 0.8em;
+}
+
+.magazine-section .card-body .card-title {
+  color: black;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 1em;
+  line-height: 5px;
+}
+
+.magazine-section .img-container {
+  height: 310px;
+}
+
+.magazine-section .sun-img {
+  margin-top: 70px;
+}
+
+section .heading-img {
+  width: 25px;
+  text-align: center;
+  margin-right: 10px;
+}
+
+.headings {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50px;
+  margin: 0;
+  padding: 0;
+  border-bottom: 1px solid lightgray;
+}
+
+.editor-pick .card .card-body .header {
+  font-size: 1em;
+  font-weight: 800;
+  margin-top: 10px;
+}
+
+.news-category .main-card img {
+  height: 230px;
+}
+
+.news-category .main-card .card-title {
+  font-size: 1.1em;
+  margin-top: 10px;
+}
+
+.news-category .hori-card-wrapper .card-body .card-title {
+  font-size: 0.9em;
+}
+
+.news-category .news-cat-head {
+  font-weight: 700;
+  font-size: 1.2em;
+  color: #f72210;
+}
+
+.news-category .horizontal-card {
+  border: none;
+}
+
+.news-category .biden img {
+  height: 250px;
+}
+
+.news-category .biden {
+  background-color: rgb(250, 250, 250);
+  border: none;
+  padding: 30px 40px;
+}
+
+.news-category .biden .card-body .card-title {
+  font-weight: 700;
+  font-size: 1.3em;
+}
+
+.news-category .news-letter .card-body .card-title {
+  font-weight: 700;
+  font-size: 1.5em;
+}
+
+.news-category .biden .card-body p {
+  font-size: 0.9em;
+}
+
+.horizontal-card .card-body .btn {
+  background-color: #f72210;
+  color: white;
+  font-weight: 700;
+}
+
+.news-category .news-letter {
+  border: 1px solid lightgray;
+  padding: 0;
+}
+
+.news-category .news-letter .card-body {
+  padding: 0 15px;
+}
+
+.news-category .news-letter .row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.news-category .news-letter .row input {
+  width: 80%;
+  font-size: 0.8em;
+  margin-bottom: 10px;
+}
+
+.news-category .biden .row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* footer section styling */
+
+footer {
+  width: 85vw;
+  margin: 0 auto;
+}
+
+footer .footer-top {
+  border-bottom: 1px solid lightgray;
+}
+
+footer .footer-brand {
+  color: #f72210;
+  font-size: 2.5em;
+  font-weight: 800;
+  margin-right: 10px;
+}
+
+footer .copy-right {
+  padding-top: 15px;
+  font-size: 0.7em;
+}
+
+footer .socials .fab {
+  color: gray;
+  padding-left: 10px;
+  font-size: 1.1em;
+}
+
+@media screen and (max-width: 790px) {
+  .main-setion-container {
+    width: 95vw;
+  }
+
+  .navbar-brand {
+    display: none;
+  }
+
+  .time {
+    display: none;
+  }
+
+  .login {
+    display: none;
+  }
```

Comment:
- [ ] This can be done with the bootstrap classes `d-*-none` (eg: `d-sm-none` or `d-md-none` depending on what screen size you need.)